test/index/fwd_decl/external_symbols.snapshot.cc

```diff
@@ -0,0 +1,13 @@
+// [..] f(49f6e7a06ebc5aa8).
+// documentation
+// | Look, a function!
+// [..] undo_magic(49f6e7a06ebc5aa8).
+// [..] perform_magic(49f6e7a06ebc5aa8).
+// documentation
+// | Abracadabra!
+// [..] C#m(49f6e7a06ebc5aa8).
+// documentation
+// | Look, a method!
+// [..] S#
+// documentation
+// | Look, a struct!
```

Comment:
FIXME: This should not be external symbols, as the symbols already exist in mydefs.cc